app-modules/ai/src/Filament/Resources/QnaAdvisors/Pages/ManageCategories.php

```diff
@@ -106,7 +107,7 @@ public function form(Schema $schema): Schema
                 Textarea::make('description')
                     ->required()
                     ->string()
-                    ->maxLength(65535)
+                    ->maxLength(CorrectQnaAdvisorCategoryDescriptionLengthFeature::active() ? 65535 : 255)
```

Comment:
Open to input from @Orrison, and if I'm wrong the work can probably be quite easily reverted, but in this case I probably wouldn't add a feature flag.

The max length is already deployed as incorrect, so for the sake of fixing it for a few minutes between the deployment and the migrations, I don't think its worth the extra work creating and doing the feature flag cleanup ticket. If it wasn't already broken, I wouldn't say that.

Reply:
I am fine with that, @rebecca-canyon go ahead and move forward with @danharrin's suggestion and remove the feature flag.

Then re-request my review when that is done. Thank you!